src/config/baselines.tsx

```diff
@@ -0,0 +1,55 @@
+import nsoDisabled from '../data/nso/NSO_Disabled_Admin1_Total.json';
+import nsoHerders from '../data/nso/NSO_Herder_HHs_Admin2.json';
+import nsoHerdsize from '../data/nso/NSO_Herd_Size_Admin1_LT_200.json';
+import nsoChild from '../data/nso/NSO_Child_U5_Admin2.json';
+import nsoLivestock from '../data/nso/NSO_Livestock_Count_ths_Admin2.json';
+import nsoElderly from '../data/nso/NSO_Single_Elderly_Admin1_Total.json';
+import nsoPoverty from '../data/nso/NSO_Poverty_Headcount_Admin1.json';
+import nsoPop from '../data/nso/NSO_Population_Admin2_Total.json';
+
+export interface NsoData {
+  TBL_ID: string;
+  Period: string;
+  CODE?: string;
+  SCR_MN: string;
+  SCR_ENG: string;
+  CODE1?: string;
+  CODE2?: null;
+  SCR_MN1: string;
+  SCR_ENG1: string;
+  SCR_MN2?: null;
+  SCR_ENG2?: null;
+  DTVAL_CO: string;
+}
+
+export interface NsoDataset {
+  DataList: NsoData[];
+}
+
+const nsoDatasets = {
+  nsoDisabled,
+  nsoHerders,
+  nsoHerdsize,
+  nsoChild,
+  nsoLivestock,
+  nsoElderly,
+  nsoPoverty,
+  nsoPop,
+};
+
+type DatasetKeys =
+  | 'nsoDisabled'
+  | 'nsoChild'
+  | 'nsoHerders'
+  | 'nsoHerdsize'
+  | 'nsoLivestock'
+  | 'nsoElderly'
+  | 'nsoPoverty'
+  | 'nsoPop';
+
+export function getNSOData(dataset: string | undefined) {
```

Comment:
This type signature is better written as `dataset?: string`.

More generally though, this function is throwing away the safety that TypeScript should be giving us. There's no reason to make this an optional parameter and then do all these checks that the compiler can make much easier than we can.

Writing the function this way:

```typescript
export function getNSOData (dataset: DatasetKeys = 'nsoPoverty') {
  return nsoDatasets[dataset];
}
```

Should give us the same result, but ensure that the compiler gives us a compile-time error if we try to look-up using an invalid key, which is really the whole point of using TypeScript.